bnd.bnd

```diff
@@ -0,0 +1,10 @@
+Bundle-Description: HiveMQ Java MQTT Client 
+Bundle-License: Apache 2.0 License
+Export-Package: \
+  com.hivemq.client.mqtt.*, \
+  com.hivemq.client.mqtt.rx.*, \
+  com.hivemq.client.mqtt.rx.reactivestreams.*, \
+  com.hivemq.client.mqtt.util.*
+Import-Package: \
+  *
+-consumer-policy: ${range;[==,=+)}
```

Comment:
Why is this necessary?

Reply:
Standard policy is `[==,+)` which would be (for netty) `[4.1,5)`, i.e. 4.1 and above, below 5. Are we sure that 4.2 (or 4.9) is fully bckward compatible? If so, we can remove that.

Reply:
I checked https://netty.io/news/2013/08/27/semantic-versioning.html and I think that the more restrictive policy I have chosen is correct (at least for netty) because different minor versions are not guaranteed to be binary compatible (which is what we need here).

I can add a version for the dagger import if you want.

Reply:
Is there any advantage of adding a version for dagger?